lib/stripe/account.rb

```diff
@@ -82,6 +82,13 @@ def serialize_params_account(_obj, update_hash)
             serialize_additional_owners(entity, owners)
         end
       end
+      if (transfer_schedule = update_hash[:transfer_schedule])
+        if (transfer_schedule[:interval] == "weekly")
```

Comment:
Just realized I should do `transfer_schedule[:interval] != "monthly"` here, and similar for weekly.